articles/native-platforms/phonegap/01-login.md

```diff
@@ -6,10 +6,10 @@ description: This tutorial will show you how to use the Auth0 Phonegap SDK to ad
 ## Phonegap Tutorial
 
 <%= include('../../_includes/_package', {
```

Comment:
@mehreencs87 Please add the `githubUrl` line. 
[contributing guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects)

Also, can you please move the System Requirements panel at the top of the doc, before this include? Thanks!